tm/src/main/java/io/seata/tm/api/TransactionalTemplate.java

```diff
@@ -97,39 +97,39 @@ private void completeTransactionAfterThrowing(TransactionInfo txInfo, GlobalTran
         }
     }
 
-    private void commitTransaction(GlobalTransaction tx) throws TransactionalExecutor.ExecutionException {
+    public static void commitTransaction(GlobalTransaction tx) throws TransactionalExecutor.ExecutionException {
```

Comment:
why public ？not use template#execute?

Reply:
Because I found that many projects use block method to execute SQL, it is unnecessary to copy the methods in the void commitTransaction again, instead of directly calling the methods of TransactionalTemplate.

Reply:
Sorry, maybe I misread this code.